specs/payment_version.md

```diff
@@ -0,0 +1,17 @@
+### Added possibility of setting version of payment protocol
+
+A new field was added to offer on provider side and demand on requestor side
+golem.com.payment.protocol.version
+There is only one possible value for now: 2
+
+```golem.com.payment.protocol.version = 2```
+
+This field is validate against constraint in the requestor demand (it is added by default by yagna):

```

Comment:
Constraint in yagna is something implementation specific. In general it shouldn't be in golem-architecture.

We can leave it for now. I will move things around when progress documentation will be merged.
But for now we should move this to last paragraph rather